ibethel/shorty
# Admin and forms 48 49

Resolves Tickets:
https://bethel-media.codebasehq.com/projects/dev-ops/tickets/49

Testing Instructions for Admin namespace:
1. Visit the normal root url.
2. You should be immediately redirected to bethel.global
3. add "/admin" to the normal root url.
4. You should be brought to the normal sign in page.
5. Sign in.
6. You should be on the main shorts index.
7. Navigate around.
8. It should all work.

Testing Instructions for Embedded Forms:
1. Navigate to the admin area for Embedded Forms (there should be a link at the top of the main admin page).
2. Create a new Embedded Form.
3. Submit various versions of the form. It shouldn't work if you don't include a name and form url. It should fill in the url slug if you don't enter it.
4. Put in a valid form url (the share URL from survey gizmo, for instance).
5. Create the form object and return to the index.
6. Click on the "Local Link".
7. It should take you to a new page with the form embedded within it (no extra styling at the moment).
